.eslintrc.js

```diff
@@ -12,17 +12,60 @@ module.exports = {
   extends: [
     'eslint-config-airbnb',
     'plugin:prettier/recommended',
-    'plugin:flowtype/recommended',
-    'prettier/flowtype',
     'prettier/react',
   ],
-  parser: 'babel-eslint',
+  overrides: [
+    {
+      files: ['*.tsx', '*.ts'],
+      parser: '@typescript-eslint/parser',
+      parserOptions: {
+        ecmaVersion: 2018,
+        sourceType: 'module',
+      },
+      extends: [
+        'plugin:@typescript-eslint/recommended',
+        'prettier/@typescript-eslint',
+      ],
+      rules: {
+        'no-useless-constructor': 0,
```

Comment:
What's the reason behind disabling this rule?

Reply:
because there is an equivalent rule `@typescript-eslint/no-useless-constructor` - I think its a bug in `@typescript-eslint` that the built in eslint rule is not disabled.

Reply:
fixed in e22711fa776f35e8c89c9156796c02bf7edcb5fb